docker-store/trustchain.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Docker Store Trust Chain
+keywords: trust, chain, store
+---
+
+## For consumers
+
+Docker ensures that all content is securely received and verified from original producers, and additionally vets images before adding them to the Docker Store.  Docker will cryptographically sign the images upon completion of a satisfactory image check, so that you can verify and trust certified content from the Docker Store.
```

Comment:
Is "vets" the right word? Maybe approves? Audits?

Can you avoid future tense?